doc/ref_model/chapters/chapter03.md

```diff
@@ -508,12 +509,74 @@ Figure 3-10 shows a simple architecture of an SFC with multiple VNFs, as SF data
+
+Figure 3-11b shows SFC creation call flow, separated logically in two steps.
+<p align="center"> <img src="../figures/ch03-model-sfc-info-create-flow.png" alt="Creation of Service Function Chain" Title="Creation of Service Function Chain" width="45%"/></p>
+<p align ="center"><b>Figure 3-11b:</b> Creation of Service Function Chain</p>
+
+1.	Creation of service functions of SFC.
+
+The flow of steps to enable the SFC creation can be as follows:-
+
+a.	SFC orchestrator creates the SFs with help of VNF MANO or CNF MANO.
+
+b.	SFC Rendere attaches the SFC aware interfaces at SFs to enable Service plane 
+
+c.	NFVO boots up the relevant SF configurations at SF.
+> **Note:** These steps are optional, if SFC orchestrator disovers that SFs are already created and existing.
+
+2.	Creation of service function path (SFP) using the created SFs and associated interfaces.
+
+A Service Function Path consists of:
+- A set of ports( in VNF environment) or interfaces ( in CNF environment) , that define the sequence of service functions 
+- A set of flow classifiers that specify the classified traffic flows entering the chain.
+
+This step creates a new chain policy with chain rules. Chain rules can include the identifier of a traffic flow, service characteristics, the SFC identifier and related information to route the packets along the chain. 
+
+Service characteristics can be application layer matching information (e.g., URL). Traffic flow identifier can be kind of traffic (e.g., Video, TCP, HTTP) flow need to be serviced. It can be specific Subscriber to apply service (e.g., parental control). The SFC identifier to steer the matched traffic along the SFP with SFC encapsulation. 
+
+a.	SFC orchestrator creates SFP with help of SDNC.
+
+b.	SDNC pushes the SFC traffic steering policies to SFF(s).
+
+c.	SFC classifier Policy provided for SFP to SFC classifier by SFC Controller. **Note:** not shown in call flow.
+
+<a name="3.5.5.3.2"></a>
+##### 3.5.5.3.2 Updation of Service Function Chain
+SFP or SFC can be updated for various reasons and some of them are :-
+- SFC controller monitors the SFP status and alerts SFC controller in case of not meeting SLA or some anamoly.
+- SFC design changes to update SF order, inclusion/removal of SFs
+- SFC Policy Rules changes
+
+<a name="3.5.5.3.3"></a>
+##### 3.5.5.3.3 Data Steering in Service Function Chain
+
+Figure 3-11c shows traffic steering along SFP.
+<p align="center"> <img src="../figures/ch03-model-sfc-data-flow.png" alt="Data steering in Service Function Chain" Title="Data steering in Service Function Chain" width="45%"/></p>
+<p align ="center"><b>Figure 3-11c:</b> Data steering in Service Function Chain</p>
+
+- SFC classifier detects the traffic flow based on classification policies. For example, to enable SGi-Lan feature as SFC, UPF acts as SFC classifier.  UPF receives the classification policies from PCF (via SMF) as traffic steering policies. 
```

Comment:
Can you please explain the acronyms at their first apperance?

Reply:
sure , i will bring full form. Accepted.